src/components/CardLists/index.js

```diff
@@ -16,17 +19,27 @@ function CardlistIndex() {
 			<StyledCardlist>
 				{exercises.map(exercise => {
 					return (
-						<StyledCard key={exercise._id}>
+						<StyledCard key={exercise.id}>
 							<StyledCardHeader>
-								<StyledHeadline>{exercise.name}</StyledHeadline>
+								<StyledCardHeadline>{exercise.name}</StyledCardHeadline>
 								<StyledCalendarButton
 									onClick={() => {
-										addCard(exercise.name);
+										addCard(
+											exercise.name,
+											exercise.image,
+											exercise.description
+										);
 									}}
 								>
 									{'add to planner'}
 								</StyledCalendarButton>
 							</StyledCardHeader>
+							<StyledCardBody>
+								<StyledCardImage src={exercise.image}></StyledCardImage>
```

Comment:
To use a` <image />` element is not wrong but I would suggest you to use the `<Image />  component of next js`, which brings some advantages. 
Here's a link to the documentation: [next/image](https://nextjs.org/docs/api-reference/next/image) 👀